0TA_Template_Sketch/0TA_Template_Sketch.ino

```diff
@@ -12,8 +15,12 @@ void setup() {
 }
 
 void loop() {
+#ifndef ESP32_RTOS && ESP32
```

Comment:
I'm not sure if this should be `#if !defined(ESP32_RTOS) && !defined(ESP32)` or `#if !defined(ESP32_RTOS) || !defined(ESP32)`. See my next comment for an explanation of the `defined()` syntax.

Reply:
I have fixed that.